### PR TITLE
Add views and basic layout

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Add views for all the required routes.
2. Implement a basic layout.
3. Create two basic users to make the app work.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+    <div class="user-name">
+      <h1>
+        <a href="/users/<%=user.id%>"><%= user.name %></a>
+      </h1>
+      <p>
+        <%= user.posts_counter%>
+      </p>
+    </div>
+  </div>
+<% end %>
```
Comment (petudeveloper) [inline app/views/users/index.html.erb:16]: [OPTIONAL] Is a good practice to use partials to keep your code DRY. Partial templates - usually just called "partials" - are another device for breaking the rendering process into more manageable chunks. With a partial, you can move the code for rendering a particular piece of a response to its own file.

```
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,90 @@
...
+      </p>
+      <div class="comments-likes">
+        <p class="comments-count">
+          Comments: X
+        </p>
+        <p class="likes-count">
+          Likes: Y
+        </p>
+      </div>
+    </div>
```
Comment (petudeveloper) [inline app/views/users/show.html.erb]: You are hard coding the recent post from the users, but you need to use your method `recent_posts_from_user`. I read your text on the `Additional Information`. In order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,90 @@
...
+  </div>
+
+  <a href="show/posts">See all posts </a>
```
Comment (petudeveloper) [inline app/views/users/show.html.erb]: When clicking on `See all posts` the link is not opening user's post index as displayed on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABQT26MCP6KHSEH6OTLM34ZWYPQQSUA).

You are hard coding the posts from the users. I read your text on the `Additional Information` and in order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1,2 +1,82 @@
...
+        <p>Username: Comment 3</p>
+      </div>
+      <div>
+        <p>Username: Comment 4</p>
+      </div>
+    </div>
+  </div>
+
+  <a href="posts/show">Pagination</a>
+</section>
```
Comment (petudeveloper) [inline app/views/posts/index.html.erb]: You are not using the method `post.recent_comments`, please use this method to render 5 most recent comments. I read your text on the `Additional Information` and in order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -1,2 +1,75 @@
-<h1>Posts#show</h1>
-<p>This is going to show all details from one post of one user</p>
\ No newline at end of file
+<section class="pagination">
```
Comment (petudeveloper) [inline app/views/posts/show.html.erb:1]: Post show page is not displaying post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png?raw=true). PLease fix this, I read your text on the `Additional Information` and in order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @Lordkaito, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] In order for your project to look more professional you can consider adding instructions for creating and migrating databases before running the rails server.  This will help the more inexperienced developers:
```
rails db:create
rails db:migrate
```

**_Note: I left a message to you on slack_**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @Lordkaito 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**